_test/test_mpi_wrappers/ClusterSlurmTester.m

```diff
@@ -0,0 +1,85 @@
+classdef ClusterSlurmTester < ClusterSlurm
```

Comment:
Can this be generalised beyond slurm to arbitrary queuing systems? Even if we're not using them right now, it would be useful to note.

Reply:
Hell it knows. Its not trivial to write completely generic code. The attempt is to achieve exactly defining the ClusterWrapper API but the differences between particular implementations are substantial.